pkg/client/client.go

```diff
@@ -838,7 +840,7 @@ func (o *OptimizelyClient) GetVariation(experimentKey string, userContext entiti
 
 // Track generates a conversion event with the given event key if it exists and queues it up to be sent to the Optimizely
 // log endpoint for results processing.
-func (o *OptimizelyClient) Track(eventKey string, userContext entities.UserContext, eventTags map[string]interface{}) (err error) {
+func (o *OptimizelyClient) Track(ctx context.Context, eventKey string, userContext entities.UserContext, eventTags map[string]interface{}) (err error) {
```

Comment:
why do we trace Track only?  Is this just a sample?

Reply:
Yes, part of a spike. We won't merge this PR. I just write code instead of a spike doc.

Reply:
Goal is to understand the implementation strategy as well as the volume of code changes and to understand the breaking changes as well.